CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- Improve clarity of verify command output.
```

Comment:
I just noticed that the changelog here does not contain tags like `added` or `fixed` etc. Maybe we should start using them.